### PR TITLE
Planner Documentation Updates - Adding Required Permissions 

### PR DESCRIPTION
**Pull request by LeonArmston**
## Type ##
- [X] Bug Fix
- [ ] New Feature
- [ ] Sample

## What is in this Pull Request ? ##
Updated all PnP Planner documentation pages to include Required Microsoft Graph Permissions and changed group parameters to accept group name and id.

### CODE REVIEW

File: documentation/Get-PnPPlannerTask.md [hunk 1]
```diff
@@ -89,7 +95,7 @@ Accept wildcard characters: False
 ```
 
 ### -Group
-Specify the group id of group owning the plan.
+Specify the group idor name of group owning the plan.
```
Comment (erwinvanhunen) [inline documentation/Get-PnPPlannerTask.md:98]: Space missing :-)

Comment (LeonArmston) [reply documentation/Get-PnPPlannerTask.md]: Fixed :-)

Can you restart the workflow to fix the commit for Set-PnPPlannerTask please?

**Comment by erwinvanhunen** [comment]
Hi! Thank you for this PR. Unfortunately the PR fails to compile. We build the internal help file for the PowerShell module based on these files and the script that builds the cmdlets failed building it. The issue is with the Set-PnPPlannerPlan.md file which lists the permissions under the syntax header instead of under synopsis.

**Comment by LeonArmston** [comment]
All fixed now and it compiles - apologies!

**Pull request merged**